CMakeLists.txt

```diff
@@ -256,3 +256,4 @@ configure_file(
 
 add_custom_target(uninstall
     COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
+
```

Comment:
Whoops. This extra line is my fault (or, well, the fault of my text editor). I'll take this line out in a pull request to your develop branch. If you accept my pull request to your branch, this pull request will be updated accordingly. Is that clear as mud?